lib/pharos/scripts/configure-cri-o.sh

```diff
@@ -14,3 +14,10 @@ systemctl enable crio
 # remove unnecessary cni plugins
 rm /etc/cni/net.d/100-crio-bridge.conf /etc/cni/net.d/200-loopback.conf || true
 systemctl start crio
+
+# Install crictl binary
+CRICTL_DOWNLOAD_SHA="597a4db0289870d81d0377396ddaf4c23725a47b33b30856e6291c2b958786f3"
+curl -sSL https://github.com/kubernetes-incubator/cri-tools/releases/download/v1.0.0-beta.0/crictl-${CRICTL_VERSION}-linux-${CPU_ARCH}.tar.gz -o /tmp/crictl.tar.gz
```

Comment:
This re-downloads/installs `crictl` for every `pharos-cluster up` run? Needs a conditional to check for the existence of `crictl`... and possibly also something for version changes?

Reply:
done